### ISSUE TITLE
Copy member always thinks I am trying to replace existing member

### ISSUE THREAD

**Issue by m-tyler** [codefori/vscode-ibmi#1729]
Copy member always thinks I am trying to replace existing member.
![image](https://github.com/codefori/vscode-ibmi/assets/1353665/d3f6f0de-cecf-45bc-965e-abb9ba592bdf)

<hr />

|Context|Version|
|-|-|
|Code for IBM i version|2.6.0|
|Visual Studio Code version|1.83.0|
|Operating System|win32_x64|

<details>
<summary>Active extensions</summary>

```
Auto Hide (fork) (vscode-autohide-fork): 1.0.8
Code Spell Checker (code-spell-checker): 3.0.1
Code for IBM i Walkthroughs (vscode-ibmi-walkthroughs): 0.4.0
Data Preview (vscode-data-preview): 2.3.0
Db2 for IBM i (vscode-db2i): 0.6.0
Emmet (emmet): 1.0.0
Error Lens (errorlens): 3.15.0
Excel Viewer (gc-excelviewer): 4.2.58
Extension Authoring (extension-editing): 1.0.0
Git (git): 1.0.0
Git Base (git-base): 1.0.0
GitHub (github): 0.0.1
GitHub Authentication (github-authentication): 0.0.2
IBM i Notebooks (vscode-ibmi-notebooks): 0.0.6
JSON Language Features (json-language-features): 1.0.0
Merge Conflict (merge-conflict): 1.0.0
NPM support for VS Code (npm): 1.0.1
Node Debug Auto-attach (debug-auto-launch): 1.0.0
Overtype (overtype): 0.5.0
Partial Diff (partial-diff): 1.4.3
RPGLE (vscode-rpgle): 0.24.0
Rainbow CSV (rainbow-csv): 3.9.0
Text Marker (Highlighter) (text-marker): 1.11.0
TypeScript and JavaScript Language Features (typescript-language-features): 1.0.0
Version Lens (vscode-versionlens): 1.9.2
WSL: Recommender (remote-wsl-recommender): 0.0.19
undefined (indent-rainbow): 8.3.1
```
</details>

<hr />

<details>
<summary>Remote system</summary>

|Setting|Value|
|-|-|
|IBM i OS|V7R5M0|
|Tech Refresh|2|
|CCSID|37|
|SQL|Enabled
|Source dates|Enabled

### Enabled features

|/QOpenSys/pkgs/bin|/usr/bin|/QSYS.lib/ILEDITOR.lib|/QSYS.LIB|/QIBM/ProdData/IBMiDebugService/bin|
|-|-|-|-|-|
|bash|attr|GENCMDXML.PGM|QZDFMDB2.PGM|startDebugService.sh|
|git|iconv|GETNEWLIBL.PGM|||
|grep|setccsid||||
|ls|||||
|md5sum|||||
|sort|||||
|stat|||||
|tar|||||
|tn5250|||||
</details>

<details>
<summary>Shell env</summary>

```bash
BUILDLIB=PGMT
CURLIB=PGMT
HOME=/home/PGMT
HOST=400devl.go.winco.local
LIBLS=QTEMP QGPL HAWKEYE WFISRC AFPTOOL SPLTOOL TAATOOL WFIOBJ WFIHST WFINEP WFIDTA DTSDCDTA DTSCNTRL DTSUSER DEVSRC DEVOBJ DEVDTA WSBSRC WSBOBJ WSBDTA DEVQUAL WSBUSR WIASPUSR D1CVTPROD D1CVTQUAL D1CVTINTG D1WFIPROD D1WFIQUAL D1WFIINTG D1DTSPROD D1DTSPRODC D1DTSPRD92 D1DTSQUAL D1DTSQUA92 D1DTSINTG D1DTSINT92 D1SYSPROD D1SYSQUAL D1SYSINTG PGMT
LMI_REL_IDLIB=D1WFIPROD
LMI_REL_IDLIBS=D1WFIPROD,WFICVT
LOGIN=pgmt
LOGNAME=pgmt
MAIL=/var/spool/mail/pgmt
OLDPWD=/home/PGMT
PATH=/QOpenSys/pkgs/bin:/QOpenSys/usr/bin:/usr/ccs/bin:/QOpenSys/usr/bin/X11:/usr/sbin:.:/usr/bin
PCR=PCR2101900
PCRNUM=2101900
PROFILENAME=D1
PWD=/home/PGMT
REL=D1WFI
SHELL=/QOpenSys/pkgs/bin/bash
SHLVL=1
SSH_CLIENT=10.10.0.135 56744 22
SSH_CONNECTION=10.10.0.135 56744 10.0.4.31 22
TZ=<MST>7<MDT>,M3.2.0,M11.1.0
USER=pgmt
USERNAME=PGMT
_=/QOpenSys/pkgs/bin/env
```
</details>

<details>
<summary>Variants</summary>

```json
{
  "american": "#@$",
  "local": "#@$"
}
```
</details>

<details>
<summary>Errors</summary>

```json
[
  {
    "command": "/QOpenSys/usr/bin/qsh",
    "code": 1,
    "stderr": "CPF9802:  Not authorized to object QRPGSRC in D1WFIQUAL.",
    "cwd": "/home/PGMT"
  },
  {
    "command": "/QOpenSys/usr/bin/qsh",
    "code": 1,
    "stderr": "CPF9815:  Member CVT21019R7 file QRPGSRC in library PGMT not found.",
    "cwd": "/home/PGMT"
  },
  {
    "command": "/QOpenSys/usr/bin/qsh",
    "code": 1,
    "stderr": "CPF9815:  Member CVT21019R7 file QRPGSRC in library PGMT not found.",
    "cwd": "/home/PGMT"
  }
]
```
</details>

**Comment by worksofliam** [comment]
Fixed in #1728 that I just opened.

**Comment by m-tyler** [comment]
Just loaded 2.6.1 and now it works as expected for me!!!

Thanks.
